go-controller/pkg/util/kube.go

```diff
@@ -16,13 +16,17 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/cert"
 
+	egressfirewallclientset "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/crd/egressfirewall/v1/apis/clientset/versioned"
+
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/cni/types"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 )
 
 // NewClientset creates a Kubernetes clientset from either a kubeconfig,
 // TLS properties, or an apiserver URL
-func NewClientset(conf *config.KubernetesConfig) (*kubernetes.Clientset, error) {
+func NewClientset(conf *config.KubernetesConfig) (*kubernetes.Clientset, *egressfirewallclientset.Clientset, *apiextensionsclientset.Clientset, error) {
```

Comment:
nit: rename this to `NewClientsets` since this method now initializes multiple client sets.

Reply:
done